Explorer/Assets/DCL/PerformanceAndDiagnostics/Diagnostics/ReportsHandling/Handlers/SceneDebugConsoleReportHandler.cs

```diff
@@ -0,0 +1,35 @@
+﻿using CommandTerminal;
+using System;
+using UnityEngine;
+using Object = UnityEngine.Object;
+
+namespace DCL.Diagnostics
+{
+    public class SceneDebugConsoleReportHandler : ReportHandlerBase
+    {
+        private readonly SceneDebugConsole sceneDebugConsole;
+
+        public SceneDebugConsoleReportHandler(ICategorySeverityMatrix matrix, bool debounceEnabled) : base(matrix, debounceEnabled)
+        {
+            sceneDebugConsole = GameObject.Instantiate(Resources.Load<GameObject>("SceneDebugConsole")).GetComponent<SceneDebugConsole>();
```

Comment:
Can we load it as an addressable?

Reply:
I will try

Reply:
I started implementing it but it gets super unnecessarily complex as usual... it involves all of the following just to avoid 1 line instantiating the resource during the app initialization:
- Configure the prefab as addressable
- Add the addressable to the Plugin settings of the `BootstrapContainer` (the one that instantiates the `DiagnosticsContainer`)
- Inject the `AssetsProvisioner` instance into the `DiagnosticsContainer` (that is easy peasy because it already exists in the `BootstrapContainer`)
- Create a different assembly for the `SceneDebugConsoleReportHandler` to be able to use the `IAssetsProvisioner` without cyclic dependencies...
- Find a way to do a fire & forget of `assetsProvisioner.ProvideMainAssetValueAsync()`

We can implement that separately if it's needed in the future